scripts/calibre-web-nginx.conf

```diff
@@ -1,7 +1,30 @@
 # RELATED: https://github.com/janeczku/calibre-web/wiki/Setup-Reverse-Proxy
 
+# 2023-12-04: This is part of an (interim?) workaround so that playback of
+# larger videos actually works.  Far better would be a proper MM (memory
+# management) solution that respects Calibre-Web's existing username/password
+# security model.  Can anybody suggest concrete ideas to make that happen?
+#
+# The failure of large videos to play is summarized here:
+# https://github.com/iiab/calibre-web/issues/37
+#
+# This (perhaps imperfect) proposed solution is emerging here:
+# https://github.com/iiab/calibre-web/pull/50
+# https://github.com/iiab/calibre-web/pull/51
+# https://github.com/iiab/calibre-web/issues/53
+# https://github.com/iiab/iiab/pull/3676
+# https://github.com/iiab/calibre-web/pull/55
+# https://github.com/iiab/calibre-web/pull/56
+# https://github.com/iiab/calibre-web/pull/57
+
 # http://box/library/calibre-web/<author>/<title>/<webm_or_mp4_video_file>
-location /library/calibre-web/ {
-    alias /library/www/html/calibre-web/;
-    fancyindex on;    # autoindex on;
+location /books-direct/ {
+    alias /library/calibre-web/;
+    index off;  # Disable automatic index file processing
+
+    # Allow serving of webm and mp4 files
+    location ~* \.(webm|mp4)$ { }
+
+    # Deny access to all other content within /books-direct/
+    location /books-direct/ {deny all;}
```

Comment:
1. `index off;` appears erroneous:
`autoindex off;` is what you mean I suspect?  Specifically:
https://nginx.org/en/docs/http/ngx_http_autoindex_module.html in contrast to...
https://nginx.org/en/docs/http/ngx_http_index_module.html which is really used for `index index.html;` etc.

   (While we investigate preserving Calibre-Web's default behaviors, as much as possible anyway!)

3. Many NGINX folks seem to prefer `location /books-direct/ {deny all;}` **on top** of the file — if only as a reminder — to make clear that [NGINX examines all prefixes first](https://serverfault.com/questions/137907/how-to-restrict-access-to-directory-and-subdirs/232706#comment1293436_138939) (before regex's, which are typically put **on the bottom** of the file, to make that clear).

4. Does `location / { deny all; }` also work?  (Or is longer-form `location /books-direct/ {deny all;}` truly required?)

```suggestion
    #autoindex off;  # Directory listings (should be off by default?)

    # Deny access to all other content
    location /books-direct/ {deny all;}

    # Allow serving of webm and mp4 files
    location ~* \.(webm|mp4)$ { }
```

Reply:
1- I meant ```autoindex off;```, thanks for catching this. And yes, it's off by default. 

2- Regarding the placement of ```location /books-direct/ {deny all;}```, it won't work for me if it's on top. 

3- Adding ```location / { deny all; }``` will result in the following error:
```
root@box:/home/iiab-admin# nginx -t
2023/12/05 15:19:23 [emerg] 2387#2387: location "/" is outside location "/books-direct/" in /etc/nginx/conf.d/calibre-web-nginx.conf:36
nginx: configuration file /etc/nginx/nginx.conf test failed
```

Reply:
> 2- Regarding the placement of `location /books-direct/ {deny all;}`, it won't work for me if it's on top. I would get a "404 not found" error.

"404 not found" error occured with which URL(s) ?

Reply:
@deldesir I'm not sure what you tested.  Please explain.

This PR works correctly as is:

(1) I tested http://box/books-direct/ == http://10.8.0.50/books-direct/ which correctly shows **403 Forbidden**.
(2) The following URL correctly plays its (tiny 256x144 rectangle!) video...

http://box/books-direct/Chubbyemu/A%20Man%20Ate%20Only%20Instant%20Noodles%20For%206%20Months.%20This%20Is%20What%20Happened%20To%20His%20Organs_%20%281%29/A%20Man%20Ate%20Only%20Instant%20Noodles%20For%206%20Month%20-%20Chubbyemu.webm

( After I clicked "Download to IIAB" and entered URL https://www.youtube.com/watch?v=fDB4TpZIgzQ )